bundles/AdminBundle/Resources/public/js/pimcore/helpers.js

```diff
@@ -2162,13 +2187,19 @@ pimcore.helpers.editmode.openVideoEditPanel = function (data, callback) {
             searchButton.disable();
             openButton.disable();
 
+            posterImageSearchButton.hide();
+            posterImageOpenButton.hide();
+
```

Comment:
Thank you for the PR! 
All works fine, ready to merge, but i would just hide/show the `posterContainer` instead of individual buttons + poster,

like this:

```
        if (type != "asset") {
            searchButton.disable();
            openButton.disable();

            poster.setValue("");
            >>>> form.getComponent("posterContainer").hide();

            form.getComponent("title").hide();
            form.getComponent("title").setValue("");
            form.getComponent("description").hide();
            form.getComponent("description").setValue("");
        } else {
            form.getComponent("title").show();
            form.getComponent("description").show();
            >>>> form.getComponent("posterContainer").show();
        }
```